README.md

```diff
@@ -10,6 +10,8 @@
 
 [Chat with the community on Discord](https://discord.gg/UxBxStPAAE)
 
+[English](/README.md) | [한국어](/docs/ko/README.md)
```

Comment:
don't forget to change the path :P

Reply:
Ah! I missed something important, thanks.